_policies/01_policies.md

```diff
@@ -59,6 +59,34 @@ Additionally, articles should include clear links to, and mention of, the releva
 For an example of an article hosted on GitHub in this style (though not a LiveCoMS article), see [the following perpetual review](https://github.com/MobleyLab/benchmarksets). 
 
 
+### Writing style and editing
+
+Articles for LiveCoMS should aim to be as clear as possible, as they are intended to aid the community, not just an expert in the field.
+In general we recommend authors follow the [ACS Style Guide](http://pubs.acs.org/isbn/9780841239999) and especially the sections on:
+- [Writing and word usage](http://pubs.acs.org/doi/pdf/10.1021/bk-2006-STYG.ch004), noting the advice to use direct, declarative sentences, often in active voice
+- [Editorial style](http://pubs.acs.org/doi/pdf/10.1021/bk-2006-STYG.ch010)
+- [Grammar, spelling, and punctuation](http://pubs.acs.org/doi/pdf/10.1021/bk-2006-STYG.ch009)
+
+We also find Plaxco's [The Art of Writing Science](http://dx.doi.org/10.1002/pro.514) to be a particularly helpful concise summary of our desired style.
+
+It is particularly important to note that **LiveCoMS does not edit articles in detail, so it is important to arrange for your own editing**.
+You may receive some comments from your editor and/or the peer reviewers that point out typos or other issues, but you should not **rely** on this for your editing process. 
+In order to keep costs to authors at a minimum, we do not employ copy editors, so be sure to arrange for your own editing.
+
+Articles can have any length; however, you should be as concise as possible.
+
+### Article layout
+
+You are responsible for preparing your article PDF and materials in the way you want them to appear in LiveCoMS, so it is important to take some care as to how these look.
+LiveCoMS will not be separately typesetting your article for you, so be careful that your materials are laid out well.
+Be sure to pay particular attention to:
```

Comment:
Probably want to emphasize that we are providing article templates, and that articles being submitted should use these templates.

Reply:
Thanks. I added a link to this effect. This is just shortly following the material where we highlight how everyone needs to use templates, so we don't need to say too much here or we'll be repetitive (but some repetition is probably a good thing).